sw/device/silicon_creator/lib/drivers/hmac.h

```diff
@@ -71,6 +110,14 @@ inline void hmac_sha256_init(void) {
   hmac_sha256_start();
 }
 
+/**
+ * Configures and starts HMAC in HMAC mode with little-endian output.
+ */
+inline void hmac_hmac_sha256_init(hmac_key_t key) {
+  hmac_hmac_sha256_configure(false, key);
```

Comment:
I think the endian should be configured as big endian.

I doubt that any other implementation of hmac is going to give us bytes in reversed order and by initializing as little endian, we're guaranteeing that we'll need to do byte reversals.

Reply:
Good point, I added an additional parameter to this function.